### PR TITLE
Updated README files for the 2021.1 release

### PR DESCRIPTION
**Pull request by x8a**

### CODE REVIEW

File: packages/node-cli/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ CLI for Node.js is an interactive tool that guides you through all the steps of
 
 ## Prerequisites
 The following software is required to work with SuiteCloud CLI for Node.js:
-- Node.js version 12.14.0 LTS or greater
+- Node.js version 14 LTS or greater
```
Comment (alii-gator) [inline packages/node-cli/README.md:15]: @vic-nik Is it safe to say use v14 or greater? New node versions might cause problems in installation etc.

**Review by alii-gator — COMMENTED** [review]
@vic-nik Shouldn't this me merged to the 21.1 release branch instead of dev?

File: packages/node-cli/README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ CLI for Node.js is an interactive tool that guides you through all the steps of
 
 ## Prerequisites
 The following software is required to work with SuiteCloud CLI for Node.js:
-- Node.js version 14 LTS or greater
+- Node.js version 14 LTS
```
Comment (alii-gator) [inline packages/node-cli/README.md:15]: @x8a Can you change the version to 14.16.0 LTS

Comment (x8a) [reply packages/node-cli/README.md]: Done!

**Review by alii-gator — APPROVED** [review]

**Pull request merged**